### ISSUE TITLE
Premium Screenshare just broke

### ISSUE THREAD

**Issue by mutschistink** [Tharki-God/BetterDiscordPlugins#102]
thats it

**Comment by RDP8** [comment]
Yeah its broken, I screenshare it works but then when I stop discord just force dcs you from the vc. Have to restart discord in order to reconnect to vc.

**Comment by Tharki-God** [comment]
Not problem with my plugin thanks and it works fine for me so idk what you doing wrong. Check with all other plugins disabled

**Comment by mutschistink** [comment]
> Not problem with my plugin thanks and it works fine for me so idk what you doing wrong. Check with all other plugins disabled

i think it doesn't work with dc bypasses 
and for some reason i only get no problems with canary

**Comment by Tharki-God** [comment]
> > Not problem with my plugin thanks and it works fine for me so idk what you doing wrong. Check with all other plugins disabled
> 
> i think it doesn't work with dc bypasses 
> and for some reason i only get no problems with canary

Which module of dc bypass does it not work with? For specific reason

**Comment by mutschistink** [comment]
> > > Not problem with my plugin thanks and it works fine for me so idk what you doing wrong. Check with all other plugins disabled
> > 
> > 
> > i think it doesn't work with dc bypasses
> > and for some reason i only get no problems with canary
> 
> Which module of dc bypass does it not work with? For specific reason

doesn't say

**Comment by Tharki-God** [comment]
> > > > Not problem with my plugin thanks and it works fine for me so idk what you doing wrong. Check with all other plugins disabled
> > > 
> > > 
> > > i think it doesn't work with dc bypasses
> > > and for some reason i only get no problems with canary
> > 
> > Which module of dc bypass does it not work with? For specific reason
> 
> doesn't say

Try and find out
